test/jasmine/tests/bar_test.js

```diff
@@ -2877,7 +2877,7 @@ describe('bar uniformtext', function() {
         })
         .then(assertTextSizes('using minsize: 9', {
             fontsizes: [12, 12, 12, 12, 12, 12, 12],
-            scales: [0, 0.48, 0.48, 0.48, 0.48, 0.48, 0.48],
+            scales: [0, 1, 1, 1, 1, 1, 1],
```

Comment:
Why did this change?

Reply:
Used to computed wrong scales as the minsize was not cleared.

Reply:
Great. Thanks!